extension.neon

```diff
@@ -1,4 +1,8 @@
 parameters:
+	nette:
+		containerLoader: null
+		applicationMapping: []
+		checkLinks: false
```

Comment:
If we're sure about the usefulness of this feature and it doesn't come with false positives, we should make it part of bleedingEdge and don't let it be optional.

Reply:
I would like to keep it optional (but might enabled by default in bleeding edge). In one of our projects we have custom overrides of some link functions in components and we need customised implementation. So i need a way to disable automatic activation of these rules to be able to use only selected rules manually

Reply:
Also this rule requires configuration either applicationMapping or contaonerLoader - without it it would output false positives. So maybe make it manually activated in any case?

Reply:
If both of these settings are not set by the user then the rule should keep quiet.

Reply:
So it would be automatically enabled by setting either of these settings AND bleedingEdge?

And could be explicitly enabled/disabled by `checkLinks` no matter if bleedingEdge is set?

Reply:
Updated